Makefile

```diff
@@ -21,17 +22,21 @@ endif
 
 
 .venv:  ## Set up virtual environment
+ifeq (, $(shell which uv))
 	python3 -m venv $(VENV)
 	$(VENV_BIN)/python -m pip install --upgrade uv
-	## Hacks to deal with grpcio compile errors on m1 macs
+else
+	uv venv $(VENV) -p $(PYTHON_VERSION)
```

Comment:
Here I choose which version of python I would use in my venv. And uv just download it and install.